tests/suites/test_suite_psa_crypto_se_driver_hal.function

```diff
@@ -565,16 +565,17 @@ static int check_persistent_data( psa_key_lifetime_t lifetime,
     psa_storage_uid_t uid = file_uid_for_lifetime( lifetime );
     struct psa_storage_info_t info;
     uint8_t *loaded = NULL;
+    int ok = 0;
```

Comment:
In such cases I'd rather use "is_ok" to show that it's a boolean, but that's an opinion.

Reply:
`int ok` is what I've used in similar support functions before, so I used the same style here for consistency.